augur/tree.py

```diff
@@ -232,8 +232,23 @@ def random_string(n):
             tmp_line = line
             if line.startswith(">"):
                 num_seqs += 1
-                for c,v in escape_dict.items():
-                    tmp_line = tmp_line.replace(c,v)
+                pos = 1
+                for i in range(1, len(line)):
+                    l = line[i]
```

Comment:
Python strings are iterable, and since the index is not needed in the loop, this can be simpler:

```suggestion
                for l in line[1:]:
```